templates/admin.html

```diff
@@ -6,6 +6,12 @@
     <link rel="shortcut icon" href="/static/eggplant_s.png">
 </head>
 <body>
-Total number of games: {{ data['game_count'] }}
+<p>Total number of games: {{ data['game_count'] }}</p>
+
+<p>All recipes: {{ data['recipes'].recipes }}
+    {% for i in data['recipes'].recipes %}
+        <div>{{i}}:</div>
```

Comment:
you're on the right path.  One weird syntax thing-- 

In jinja (the flask templating language)
`data['recipes'].recipes`
resolves to
`data['recipes']['recipes']`
which you're taking advantage of here.  It makes it almost like javascript's object syntax.

each recipe's data in your loop is then `data['recipes']['recipes'][i]`